### PR TITLE
feat: Rating component added

### PR DESCRIPTION
**Pull request by Meghana-sk**
- Added rating component
- Indentation automatically fixed by Prettier extension. Previously prettier was not used
csb link: https://6gugn2.csb.app/components/rating/rating.html

### CODE REVIEW

File: components/grid/grid.html [hunk 1]
```diff
@@ -1,107 +1,132 @@
...
+        <div class="iframe-container">
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid%2520grid-col-2%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522grid-item%2522%253EA%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522grid-item%2522%253EB%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A"
+            style="
+              width: 403px;
+              height: 262px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
```
Comment (NamrataRaikwar2002) [inline components/grid/grid.html]: you should remove these inline styling.

**Review by NamrataRaikwar2002 — APPROVED** [review]
All are looking good , good naming for image 'alt'  values. One suggestion is you should remove inline styling in iframe from all files.

File: components/alert/alert.css [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
+  width: 80%;
+  padding: 0.8rem;
+  gap: 0.4rem;
+  color: whitesmoke;
+  border-radius: 5px;
 }
 
 .alert-error {
-    background-color: var(--error-red);
+  background-color: var(>Text utiltieserror-red);
```
Comment (GauravDe-git) [inline components/alert/alert.css]: Here, the error alert is barely visible. You can try changing the colors of the error alert component.
image

File: components/colorPalette/colors.css [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
-    --whitesmoke-color: whitesmoke;
-    --dark-theme-bg-color: #121212;
-    --secondary-dark-blue: #1d2d44;
-    --secondary-dark-blue-hover: #33415c;
-    --secondary-dark-blue-disabled: rgb(29, 45, 68, 0.39);
-    --success-green: #268541;
-    --white-color: white;
-    --lightish-gray: rgb(66, 62, 62);
+  --red-color: red;
+  --online-green: rgb(18, 172, 18);
```
Comment (GauravDe-git) [inline components/colorPalette/colors.css]: As a good practice in CSS, you can use 1 type of color code. You can change these RGB values to Hex codes like the other ones.

**Review by GauravDe-git — APPROVED** [review]
I have suggested some changes. You may look into them.

File: components/colorPalette/colors.css (hunk 3 above)
Comment (Meghana-sk) [reply components/colorPalette/colors.css]: Hey Gaurav. Thank you for the suggetsion. Will change rgb() to HEX code wherever alpha value is not used

**Pull request merged**